geotrek/api/v2/filters.py

```diff
@@ -500,10 +500,14 @@ def filter_queryset(self, request, queryset, view):
             dates_after = request.GET.get('dates_after')
             if dates_after:
                 dates_after = datetime.strptime(dates_after, "%Y-%m-%d").date()
+                qs = qs.filter(Q(end_date__gte=dates_after))
             else:
                 # Filter out past events by default
                 dates_after = date.today()
-            qs = qs.filter(Q(end_date__gte=dates_after))
+                qs = qs.filter(
+                    Q(end_date__gte=dates_after) | 
+                    Q(end_date__isnull=True) & Q(begin_date__gte=dates_after)
+                )
```

Comment:
Add tests
- Events with no end_date are returned in APIv2 with begin_date as value